### ISSUE TITLE
MixItUp takes control of all buttons on page (even on links)

### ISSUE THREAD

**Issue by leomat** [patrickkunka/mixitup#564]
Imagine you have on simple collapse menu on your page:
<ul>
<li>
<a href="#" class="shop-sidebar-subtitle" data-toggle="collapse" aria-expanded="false">Click here to expand</a>
<ul class="collapse  list-unstyled" id="submenu1">
<li>
<a href="">Subcat1</a>
</li>																									<li>
<a href="">Subcat1</a>
</li>																																															
</ul>
</li>
</ul>

When you click on the "Click here to expand", the MixItUp plugin believes it's its own control button.
After investigation, it appears that this "bug" happens only when an attribute "data-*" is set on the clickable element. 

I also tried on the demo files included in the zip downloaded on Github and it happens too. 

Sorry for my English.

**Comment by MichaIng** [comment]
Create a (div) class that only holds the selectors and elements that you want to mix. Then create MixItUp with local scope on that class only:
```
	mixitup('#mixitup', {
		controls: { scope: 'local' },
	});
```
with something like this
```
<div id="mixitup">
	<button data-filter=".abc">Mix button</button>
	...
	<div class="mix abc">Mix element</div>
</div>
```
So only elements only within this `mixitup` div are considered being mixer selectors or elements.

If you want multiple mixers, you can create it multiple times each for one local scope.

**Comment by leomat** [comment]
I will try that. 

Thanks for the detailled answer !

**Comment by rakibul111** [comment]
I have tried this, its working. Thank you!

**Comment by mikexwolf** [comment]
I put the whole thing under an article with id="mixitup"

![image](https://user-images.githubusercontent.com/14350603/154532075-05a80e59-1c64-4a46-9910-b162dbe12951.png)

but now I am not sure where to put this code:

        mixitup('#mixitup', {
		controls: { scope: 'local' },
	});

Sorry for the noob question :)

**Comment by MichaIng** [comment]
In the JavaScript where you load MixItUp, see also: https://github.com/patrickkunka/mixitup#creating-a-mixer

**Comment by mikexwolf** [comment]
Thanks! It works :)

For future lost souls, here's what I did.

This : 
![image](https://user-images.githubusercontent.com/14350603/154535794-3251c9c9-6dff-4674-9d92-ee11fef3f6b7.png)

Turns into this:

![image](https://user-images.githubusercontent.com/14350603/154535766-ce753593-e582-46a0-9bca-52b5136b8c47.png)

This is the bottom of the HTML page, not the minimized JS file (which you don't want to make changes to)

**Comment by MichaIng** [comment]
Yes makes sense. I personally load it from an own JavaScript where other modules, like a slider and some scrolling/button behaviour are loaded as well. But if nothing else is done, inline JavaScript should be fine, of course.
